### ISSUE TITLE
computeLRF gives a distance 0 when it shouldn't

### ISSUE THREAD

**Issue by AliLopSan** [DessimozLab/pylabeledrf#2]
Hello!
I have two trees, which are clearly different (topologically and label-wise)
**Tree 1**: (((6,9)speciation,((13,14)duplication,12)speciation)speciation,2)speciation
**Tree 2**: (2,(9,(((13,14)duplication,12)speciation,6)speciation)duplication)speciation

When I compute LRF and ELRF the distance given by both functions is 0. 

**Tree 1** looks like this:
![tree_189_infered](https://github.com/user-attachments/assets/daa27b47-8ac9-47fd-9a32-4d187dd3a0ea)
And **Tree 2** looks like this:
![tree_189_ogt](https://github.com/user-attachments/assets/57753ef8-9830-4fc7-8ce3-efdaa3f99782)

Note that **tree 1** has 1 duplication while **tree 2** has 2 and also the placement of the leaves 9 and 6 is different in both trees, so I believe that the distance should be different from 0. However, $\texttt{computeLRF( )}$ and $\texttt{computeELRF( )}$ both give 0 as output.

I believe that the issue is that both of my trees are rooted and it seems that the function is designed for unrooted trees. I'm not so sure about this. Do you have any suggestions for this?

**Comment by alpae** [comment]
Hi @AliLopSan 

we looked into your issue, but for us the computed difference between tree1 and tree2 is 2. While investigating what could be the cause we identified that in the release on pypi (which unfortunately was outdated 🙈), there is no check in place which ensures that the trees use a shared taxon_namespace. if the trees were loaded without explicitly passing a taxon_namespace attribute, the returned LRF was indeed 0 (because it assumed that there are no taxa in common). 

we now also made a new release 0.3.2 available on pypi which checks that it's a common taxon_namespace or raises an error otherwise. In this release, the LRF is also computed in a more efficient way. 

```
taxa = dendropy.TaxonNamespace()
t1 = dendropy.Tree.get_from_string(tree1, taxon_namespace=taxa)
t2 = dendropy.Tree.get_from_string(tree2, taxon_namespace=taxa)
computeLRF(t1,t2)
```
--> 2

to upgrade to the newest version of pylabeledrf run:

```
pip install --upgrade pylabeledrf
```

Best wishes 

Adrian
